### PR TITLE
Some automated CS fixes

### PR DESCRIPTION
**Pull request by Nyholm**
They are all safe ;)

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -7,13 +7,14 @@ $config = PhpCsFixer\Config::create()
         'array_syntax' => ['syntax' => 'short'],
         'declare_strict_types' => false,
         'concat_space' => ['spacing'=>'one'],
-        // 'ordered_imports' => true,
+        'ordered_imports' => true,
```
Comment (Nyholm) [inline .php_cs]: This is just because we want it to look good

File: .php_cs [hunk 2]
```diff
@@ -7,13 +7,14 @@ $config = PhpCsFixer\Config::create()
...
         // 'phpdoc_align' => ['align'=>'vertical'],
-        // 'native_function_invocation' => true,
+        'native_function_invocation' => true,
```
Comment (Nyholm) [inline .php_cs]: This is actually a security issue

File: .php_cs [hunk 3]
```diff
@@ -7,13 +7,14 @@ $config = PhpCsFixer\Config::create()
...
     ])
     ->setFinder(
         PhpCsFixer\Finder::create()
             ->in(__DIR__.'/src')
+            ->in(__DIR__.'/tests')
```
Comment (Nyholm) [inline .php_cs]: This should be enabled because we do care about tests too

**Comment by Nyholm** [comment]
I could not use native function calls because we are actually overwriting `curl_multi_setopt` and `curl_setopt_array` in our tests

**Comment by sagikazarmark** [comment]
> They are all safe ;)

Promise? 😄

**Comment by Nyholm** [comment]
Promise!

**Pull request merged**